FFM-8566, FFM-8567: adds erlang sdk relnotes 2.0.0/2.0.1 (#2331)
harness/developer-hub: release-notes/feature-flags.md

This release does not include early access features.

##### Fixed issues

###### Feature Flags SDKs

The **Erlang** server SDK has been updated to version **2.0.0** with the following updates.

:::caution
The following are breaking changes for Erlang applications only.  

* Due to a new dependency on a murmur3 hashing library implemented in Elixir, the following is now required to use the SDK in Erlang applications:

  * Elixir version 1.13.4 or above must be installed on your build system when compiling your application.

  * Rebar3 `rebar_mix` must be installed in your Rebar3 plugins.

  For full details, go to [Install the SDK for Erlang applications](/docs/feature-flags/ff-sdks/server-sdks/erlang-sdk-reference/#for-erlang-applications).

This update does not affect Elixir applications, and no further action is required for Elixir applications upon upgrading to this version of the SDK.
:::

These changes are part of issue FFM-8364:

* Enhancement: Implemented retry logic for authentication, polling, and metrics services for resilience and fault tolerance.
* Enhancement: Changed supervisor restart intensity from 1 restart in 5 seconds to 4 restarts in 5 seconds.

> * The murmur3 nif library was giving unpredictable runtime behavior. This has been fixed by using a pure Elixir implementation.

Reviewer: > The murmur3 nif library was giving unpredictable runtime behavior. This has been fixed by using a pure Elixir implementation.

It turns out this wasn't the case, but we assumed it was at the time, and the actual issue was resolved in 2.0.1. Should it be left in any way, or removed, or altered? 

if alteration is the best solution then I would change it to :

```suggestion
* The murmur3 nif library has been replaced by pure Elixir library.
```
